external/dynamo/generalized/frontend.py

```
@@ -0,0 +1,445 @@
+            if tok.pad_token is None:
+                tok.pad_token = tok.eos_token
+            self.tokenizers[model] = tok
+        return tok
+
+    def _messages_to_text(self, messages: list[dict[str, str]], tokenizer) -> str:
+        # Try chat template first; fall back to a plain transcript
+        if getattr(tokenizer, "chat_template", None):
+            try:
+                return tokenizer.apply_chat_template(messages, tokenize=False, add_generation_prompt=True)
+            except Exception:
+                pass
```

Ruff (0.14.8)

`try`-`except`-`pass` detected, consider logging the exception

(S110)